src/cli_callback.c

```diff
@@ -543,8 +470,11 @@ static int download_file(storj_env_t *env, char *bucket_id,
         progress_cb = file_progress;
     }
 
+    // TODO: expose buffer size as user option or something
     storj_download_state_t *state = storj_bridge_resolve_file(env, bucket_id,
-                                                              file_id, fd, handle,
+                                                              file_id, fd,
+                                                              0,
```

Comment:
is it an issue for buffer size to be 0?

Reply:
If the size is 0 the default is used (`STORJ_DEFAULT_BUFFER_SIZE`).